crates/matrix-sdk-indexeddb/src/crypto_store/mod.rs

```diff
@@ -487,6 +491,13 @@ impl IndexeddbCryptoStore {
             );
         }
 
+        if let Some(i) = &dehydration_pickle_key {
```

Comment:
This doesn't seem to match to the documentation? There doesn't seem to be a way to delete the pickle key.

Reply:
yes I spotted that problem when doing the wasm test. Trying to see how to do that properly

Reply:
An enum with three variants could do the trick, something like `JsOption`.

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/4383/commits/c66a8fc608dd342c5a62c31cdd8de0608cd801d8